src/L1/system-contract/SystemSignerRegistry.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.24;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @dev Example "SystemSignerRegistry" storing `(startBlock, signer)` pairs.
+ *      The getSigners() function returns parallel arrays for block numbers and addresses,
+ *      
+ */
+contract SystemSignerRegistry is Ownable {
+    struct Signer {
+        uint64 startBlock;
```

Comment:
Didn't we decide against having the `startBlock` here?

Reply:
iirc we discussed it [here](https://scrollco.slack.com/archives/C02PKAQ2430/p1735815568292809?thread_ts=1735809529.760869&cid=C02PKAQ2430) but didn't reach a conclusion

Reply:
>  The reason is that otherwise the zkproof must accept any of two valid keys

Which zkproof do you mean? If you mean when we finalize bundles, currently we don't prove anything about the block signatures.

Reply:
Ah, I see. Let me put it back with one signer then and forget about this orderly switching with blockNum for now (bringing it back in the future probably).